guide/src/installation.md

```diff
@@ -0,0 +1,63 @@
+# Installation
+
+## Install from package managers
+
+### PyPI
+
+maturin is published as Python binary wheel to PyPI, you can install it using pip:
```

Comment:
Not sure that users know what a "binary wheel" is (it's kinda part of this guide's job to explain that!), it might be worth skipping during the installation instructions:

```suggestion
maturin is published to PyPI, you can install it using pip:
```